java-checks/src/main/java/org/sonar/java/checks/CheckList.java

```diff
@@ -20,7 +20,7 @@
 package org.sonar.java.checks;
 
 import com.google.common.collect.ImmutableList;
-
+import java.util.List;
```

Comment:
Can you please remove this useless modification from the PR ? otherwise it will be modified by Michael or my formatter for now reason in next commit. This pollutes the commit diff for nothing.